src/Entity/BaseEvent.php

```diff
@@ -101,6 +101,13 @@
      */
     protected $description;
 
+    /**
+     * @var string
+     *
+     * @ORM\Column(length=50)
+     */
+    protected $timeZone;
```

Comment:
Does this need to be exposed in apis ?
Does this need to be synced with api.en-marche.fr ?

If yes, you should add annotations regarding serialization (JMS\Group & SerializedName, you can check the beginAt & finishAt properties for examples.

Reply:
expose in api Added